### DISCUSSION TITLE
Reuse page if it exists in stack

### DISCUSSION THREAD

**Discussion by FarhanSajid1** [jonataslaw/getx#2326] [Q&A]
Currently, I see only two options when navigating to a page, we can push to the stack or we can delete from the stack. 

```
    if (previousIndex == 1) {
      Get.offNamed(pages[index], id: 1);
    } else {
      Get.toNamed(pages[index], id: 1);
    }
```

If there a way to check the history of the stack and use that ppage instead of pushing new pages to the top of the stack?

**Comment by shang1219178163** [comment]
@FarhanSajid1  look here! https://pub.dev/packages/route_stack_manager
